app/src/main/scala/org/alephium/explorer/persistence/schema/InputSchema.scala

```diff
@@ -37,13 +37,19 @@ object InputSchema extends SchemaMainChain[InputEntity]("inputs") {
     def mainChain: Rep[Boolean]           = column[Boolean]("main_chain")
     def inputOrder: Rep[Int]              = column[Int]("input_order")
     def txOrder: Rep[Int]                 = column[Int]("tx_order")
+    def outputRefTxHash: Rep[Option[Transaction.Hash]] =
+      column[Option[Transaction.Hash]]("output_ref_tx_hash")
+    def outputRefAddress: Rep[Option[Address]] = column[Option[Address]]("output_ref_address")
+    def outputRefAmount: Rep[Option[U256]] =
+      column[Option[U256]]("output_ref_amount", O.SqlType("DECIMAL(80,0)")) //U256.MaxValue has 78 digits
 
     def pk: PrimaryKey = primaryKey("inputs_pk", (outputRefKey, blockHash))
 
-    def blockHashIdx: Index    = index("inputs_block_hash_idx", blockHash)
-    def inputsTxHashIdx: Index = index("inputs_tx_hash_idx", txHash)
-    def outputRefKeyIdx: Index = index("inputs_output_ref_key_idx", outputRefKey)
-    def timestampIdx: Index    = index("inputs_timestamp_idx", timestamp)
+    def blockHashIdx: Index       = index("inputs_block_hash_idx", blockHash)
+    def inputsTxHashIdx: Index    = index("inputs_tx_hash_idx", txHash)
+    def outputRefKeyIdx: Index    = index("inputs_output_ref_key_idx", outputRefKey)
```

Comment:
Please correct me if I'm wrong but I don't think `outputRefKeyIdx` is needed because this column is already indexed in composite primary key `pk`.

Reply:
yes it's an older commit, I created #283 to tackle it, as it will require a migrations to remove the index, maybe let's try to remove more at once in one single PR